models/epidemic/epidemic.cpp

```diff
@@ -195,11 +195,6 @@ int main(int argc, const char** argv) {
     token = buffer.substr(0, pos);
     float prob_urv = std::stof(token);
     buffer.erase(0, pos + delimiter.length());
```

Comment:
remove this line

Reply:
yeah makes sense; also line 196 should be `float prob_urv = std::stof(buffer); // instead of token` i guess.